web/turkey/turkey/settings.py

```diff
@@ -131,3 +133,10 @@
         'rest_framework.parsers.JSONParser',
     )
 }
+
+# CORS
+CORS_URLS_REGEX = r'^/survey/api/.*$'
```

Comment:
Allow CORS only on the api endpoints for survey. There's no valid reason for websites to be redirected browser traffic to any other part of our site.